lib/cloudantBaseService.ts

```diff
@@ -97,13 +97,9 @@ export default abstract class CloudantBaseService extends BaseService {
    * Configuration values to use Cloudant service.
    * @param {Authenticator} userOptions.authenticator CouchdbSessionAuthenticator object can be used
    *   to authenticate requests to the service.
-   * @param {string} [userOptions.jar] When CouchdbSessionAuthenticator is the Authenticator, a Cookie Jar
-   *   must be in use. So a new jar will be created, if there is no custom one.
```

Comment:
This param is still valid, so we should not remove it from the typedoc, just reword it.

Something like:
```suggestion
   * @param {string} [userOptions.jar] A Cookie Jar. A new jar is created if there is no custom one.
```

Reply:
What about "A Cookie Jar. Enabled by default if there is no custom one." Could swap "enabled" with "created"

Reply:
That's even better!